tests/integration/test_webhook_integration.py

```
@@ -0,0 +1,274 @@
+from gh_pr.webhooks.events import EventType, WebhookEvent
+
+
+class TestWebhookIntegration(AioHTTPTestCase):
+    """Integration tests for webhook system."""
+
+    async def get_application(self):
+        """Create test application."""
+        # Create webhook config
+        self.config = Mock()
+        self.config.port = 8080
+        self.config.secret = "test_secret_key"
```

Ruff (0.13.1)

Possible hardcoded password assigned to: "secret"

(S105)